doc/appendix-de-kassensichv/cash-register-integration/cash-register-integration.md

```diff
@@ -64,15 +742,278 @@ The transaction number, defined in TR-03153, is responded behind the hash-tag in
 
 *Explicit Flow - End Transaction (DE - KassenSichV)*
 
-#### The fiskaltrust.SecurityMechanism implicit flow
+  </p>
+</details>
 
-The regular workflow of the fiskaltrust-SecurityMechanism in the German market for actions running for a short period has the same requirements as long-running ones. There has to be a "Start-Transaction" and a "Finish-Transaction" executed against the TSE. In order to speed up these two steps into one call to the 'Sign' method, a special 'ReceiptCaseFlag' is used. Each time this is used in combination with a usual 'ReceiptCase', a "Start-Transaction" is done behind the scenes upfront the final call, using the given 'ReceiptCase'.
-Using a unique identifier in 'cbReceiptReference' that was already used with a 'Sign' call with 'ReceiptCase' "Start-Transaction" will end up in an exception.
-The up-counting transaction number defined in TR-03153 is responded behind the hash-tag in the property 'ftReceiptIdentification' of 'ReceiptResponse', prefixed by "IT".
+  </p>
+</details>
 
-![implicit-flow-start-finish-transaction](media/implicit-flow-start-finish-transaction.png)
+#### Examples
```

Comment:
I'd add a short paragraph about _why_ we have this examples (e.g. what to look for, ...), and a link to https://middleware-samples.docs.fiskaltrust.cloud/ for more